docs/usage.md

```diff
@@ -112,7 +112,8 @@ to the pytest.
 * `--csv-change` - Allows changes in the OCS CSV. For example, usage of custom image,
    like MCG or RHCS. The format should be:
    <pattern_to_replace_from::pattern_to_replace_to>, while '::' is the delimiter
-
+* `--dev-mode` - Runs in development mode. Skip the checks like collecting
+   cluster versions, collection ocs versions, health checks etc.
```

Comment:
Please keep one empty line before the next `## Examples` section

Reply:
Done